.github/workflows/remove-label-on-approve.yml

```
@@ -1,26 +1,37 @@
     steps:
+      # Checkout the repository
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+
       - name: Remove multiple labels using github-script
         uses: actions/github-script@v7
         with:
+          github-token: '${{ secrets.BOT_TOKEN }}'
           script: |
             const labelsToRemove = ['waiting-review','ready to review']; // Labels to remove
             
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)